src/routes/console/project-[project]/messaging/providers/store.ts

```diff
@@ -20,6 +20,7 @@ type ProvidersMap = {
             [key in Providers]?: {
                 imageIcon: string;
                 title: string;
+                link: string;
```

Comment:
@wess, how come the link was added here?

Reply:
I can remove it, wasn't sure if we would need to reference it anywhere else.

Reply:
I'll remove it. thanks for the clarification!